### ISSUE TITLE
Delete Reports from the Task

### ISSUE THREAD

**Issue by EduRemix** [Superalgos/Superalgos#227]
When editing the product code of an indicator, it is necessary to delete the corresponding "Reports" folder before starting again the indicator Task from the designer. It is proposed to have the option of "Delete Reports" from the Task itself.

**Comment by Luis-Fernando-Molina** [comment]
Yes, this will be at the Data Product object, that represents the Data at the File System
